### PR TITLE
updated simplecalculator.cs

### PR DESCRIPTION
**Pull request by ekottifiok**
I replace all `throw new NotImplementedException();` in the `SimpleCalculator<T>` class with the proper implementation details. Run unit tests in the solution via your IDE or `dotnet test` in shell. This step is complete when all tests pass.

### CODE REVIEW

File: .vs/learn.net/v17/.wsuo
Comment (dystopiandev) [inline .vs/learn.net/v17/.wsuo:1]: -1

Delete this file.

File: .vs/VSWorkspaceState.json
Comment (dystopiandev) [inline .vs/VSWorkspaceState.json:1]: -1

Delete this file.

**Review by dystopiandev — CHANGES_REQUESTED** [review]
Resolve comments

**Comment by ekottifiok** [comment]
removed the files

File: .vs/VSWorkspaceState.json
Comment (ekottifiok) [reply .vs/VSWorkspaceState.json:1]: done

**Review by dystopiandev — APPROVED** [review]
Looks good :)

**Pull request merged**